app/scripts/lib/setupAccountLabelsPetnamesBridge.ts

```diff
@@ -0,0 +1,105 @@
+import { NameController, NameType } from '@metamask/name-controller';
+import { cloneDeep } from 'lodash';
+import log from 'loglevel';
+
+export type IdentityEntry = {
+  address: string;
+  name: string;
+};
+
+export type PertinentState = {
```

Comment:
As we're exporting this type, should we give it more context so it's meaningful in other files without an alias?

Such as `PreferencesControllerState`?